test/verify/check-storage-nfs

```diff
@@ -181,7 +181,7 @@ class TestStorageNfs(storagelib.StorageCase):
             def check():
                 choices = self.dialog_combobox_choices("remote")
                 return len(choices) == 2 and "/home/foo" in choices and "/home/bar" in choices
-            self.retry(None, check, None)
```

Comment:
If you're going to the trouble of adding default values for the arguments then you should probably go ahead and omit the `, None, None` here, no?